app/uk/gov/hmrc/tradergoodsprofiles/services/RouterService.scala

```diff
@@ -67,13 +72,61 @@ class RouterServiceImpl @Inject() (
           )
           Left(
             ServerErrorResponse(
-              dateTimeService.timestamp,
+              uuidService.uuid,
               s"Could not retrieve record for eori number $eoriNumber and record ID $recordId"
             ).toResult
           )
         }
     )
 
+  override def removeRecord(eoriNumber: String, recordId: String, actorId: String)(implicit
+    hc: HeaderCarrier
+  ): EitherT[Future, Result, Unit] =
+    EitherT(
+      routerConnector
+        .put(eoriNumber, recordId, actorId)
+        .map {
+          case httpResponse if is2xx(httpResponse.status) => Right(())
+          case httpResponse                               => Left(handleError(httpResponse.body, httpResponse.status, eoriNumber, recordId))
+        }
+        .recover { case ex: Throwable =>
+          logger.error(
+            s"[RouterServiceImpl] - Exception when removing record for eori number $eoriNumber and record ID $recordId, with message ${ex.getMessage}",
+            ex
+          )
+          Left(
+            ServerErrorResponse(
+              uuidService.uuid,
```

Comment:
correlationId ID here has not meaning and it will only create confusion for the user or user support team as this is used to track the request downstream (to EIS), but in here we haven\t set the request to EIS yet. Also this will conflit the correlationID from EIS